components/tidb_query/src/rpn_expr/mod.rs

```diff
@@ -109,7 +110,8 @@ fn divide_mapper(lhs_is_unsigned: bool, rhs_is_unsigned: bool) -> RpnFnMeta {
 }
 
 #[rustfmt::skip]
-fn map_pb_sig_to_rpn_func(value: ScalarFuncSig, children: &[Expr]) -> Result<RpnFnMeta> {
+fn map_pb_fn_to_rpn_func(expr: &Expr, children: &[Expr]) -> Result<RpnFnMeta> {
```

Comment:
`map_pb_expr_node_to_rpn_func`

And if `expr` is passed, then there is no need for a standalone `children` parameter, since it can be retrieved from `expr`.

Reply:
Nice catch